### ISSUE TITLE
APK build variations.

### ISSUE THREAD

**Issue by Erquint** [commetchat/commet#385]
The releases contain two different apk files.  
What is the difference of the one with `google-services` in the filename?  
And which one should be normally installed?

**Comment by Airyzz** [comment]
Which one you use depends on your preference, but the main difference at the moment is just how push notifications are handled. The `google-services` build uses Google's Firebase Cloud Messaging service to deliver push notifications, which is the standard way for android apps to receive notifications. 

However, some users prefer to not have any data sent to google services, and so the other build uses Unified Push to deliver notifications.

**Comment by Erquint** [comment]
Thanks for clarifying.  
That's probably worth mentioning either in the README or in the release notes.  

I gave the client a try (the `google-services` version) but it sure is in a very raw stage of development yet to really be used.  
The foremost problem for me was perhaps how session cross-signing wouldn't stick after successfully verifying and would lead to my account being shown as using unverified session to other Matrix users, which is why I had to log out of it entirely for now until further development progress.
